src/main.ts

```diff
@@ -397,14 +396,17 @@ var LZString = {
       case 2:
         return "";
     }
-    dictionary[3] = c;
+    /** TODO - address during code cleanup */
+    // @ts-ignore
+    dictionary[3] = c as string;
+    // @ts-ignore
     w = c;
+    // @ts-ignore
     result.push(c);
```

Comment:
Not a 1:1 line but this can be cleaned significantly by making it a single line, what do you think? -

```suggestion
    result.push(dictionary[3] = w = String(c));
```